clang/test/Modules/non-affecting-module-maps.m

```diff
@@ -1,6 +1,9 @@
 // RUN: rm -rf %t && mkdir %t
 // RUN: split-file %s %t
 
+// XFAIL: *
+// rdar://101268970
```

Comment:
Why does this test fail?

Reply:
Because it checks that non-affecting module maps are not being serialized into the PCM at all:
```
// CHECK-B-NOT: Input file: {{.*}}b.modulemap
```